src/app/page.js

```diff
@@ -101,11 +15,9 @@ function FeaturedProducts({ products }) {
         {/* Placeholder product cards */}
         {products.map((product) => (
           <div key={product.id} className="border rounded-lg p-4 shadow-sm bg-white">
-            <div className="h-40 bg-gray-200 rounded mb-4">
-              <img src="https://picsum.photos/200/300" alt="randomly generated media" />
-            </div>
```

Comment:
Refactored this into one line
- Implemented the Tailwind image background class to make it fit in the image container on the card.

  ```
  bg-[url(https://picsum.photos/200/300)]
  ```


[Tailwind Background Class](https://tailwindcss.com/docs/background-image)